write-api-service/src/main/java/io/ga4gh/reference/api/GitHubBuilder.java

```diff
@@ -100,7 +101,8 @@ public boolean createRepo(String organization, String repo) {
             // was not able to create the repo
             return false;
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            LOG.error(e.getMessage());
+            throw new RuntimeException("Could not create GitHub repository.  Check your GitHub token.");
```

Comment:
When throwing a runtime exception, you can pass along the original exception as well in the constructor, this avoids having to call LOG.error separately